lms/static/js/search/app.js

```diff
@@ -0,0 +1,35 @@
+
+var edx = edx || {};
```

Comment:
Oh, interesting. I just tried running that statement in Chrome, Firefox, and Safari, and it seems to work correctly even when `edx` is undefined. Today I learned something new about Javascript. :)